### PR TITLE
ANFOG_RT/DM pipelines

### PR DESCRIPTION
**Pull request by danfruehauf**
Scripts to process bundle of files uploaded as zip archive for
both ANFOG_RT and ANFOG_DM.

### CODE REVIEW

**Comment by danfruehauf** [comment]
@bpasquer Added another commit which refactors things a bit. There is a new function `nc_list_variables` which then we can `grep` on to find variables by regex.

File: ANFOG/DM/anfog_dm_netcdf_compliance.sh [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+        if [ x"$var" != x ]; then
+            nc_set_att -a long_name,$var,o,c,"${var,,} informations" $nc_file
+        fi
+    done
+}
+
+fix_depth_min_max() {
+    nc_file=$1; shift
+    # val.nc WTF?!
+    local depth_min=`ncap2 -O -C -v -s "val=DEPTH.min();print(val)" ${nc_file} ~/val.nc | cut -f 3- -d ' ' ;`
```
Comment (danfruehauf) [inline ANFOG/DM/anfog_dm_netcdf_compliance.sh]: @bpasquer I'm not sure what `~/val.nc` should refer to.

Comment (bpasquer) [inline ANFOG/DM/anfog_dm_netcdf_compliance.sh]: this is a compulsory argument -output- of the ncap2 function.  I should add a line to delete straight after extracting the min max value. we can rename it 'tmp' if you prefer

Comment (danfruehauf) [inline ANFOG/DM/anfog_dm_netcdf_compliance.sh]: Perhaps you should then:
 * Create a temporary file
 * Output to it
 * Delete it

Comment (danfruehauf) [inline ANFOG/DM/anfog_dm_netcdf_compliance.sh]: Or you can also output to `/dev/null`.

**Pull request merged**